updated Partner Showroom and amended Fees (#269)
chain4travel/camino-docs: docs/camino-messenger/introduction.md

In detail:

- Supported Services (full package name including the version, rack-rate flag, fee, capabilities) -- for suppliers to specify which services they support, whether they have a rack-rate or public rate that any Camino Partner can book without prior agreement, what fee they charge to respond to (search) messages and to specify particularities of their integration in capabilities.
- Wanted Services (full package name including the version) -- for distributors to specify which services they support and search for when matching with others
- Payment info (supported currencies: Off-Chain, CAM, ERC20 tokens, stable coins)
- Public-Keys for encrypting the private data of the booking tokens
- Active Bots for identification

### Fees

The following fees have to be paid to be able to exchange messages using the Camino Messenger:

1. Camino Messenger Network Fee

>    The Network Fee goes towards the operators of the Messenger Server. Initially the fee is set to 0.01 CAM per message. The fee is the exact same on each server on the network and will be decided by the server operators through voting. The network fee is split between the operator (70%) and Chain4Travel (30% for further development). Instead of working with an anonymous messenger server, we encourage suppliers to host their own messenger server to earn the network fee themselves.

Reviewer: ```suggestion
1. Camino Messenger Network Fee

   The Network Fee goes towards the operators of the Messenger Server. Initially the fee is set to 0.01 CAM per message. The fee is the exact same on each server on the network and will be decided by the server operators through voting. The network fee is split between the operator (70%) and Chain4Travel (30% for further development). Instead of working with an anonymous messenger server, we encourage suppliers to host their own messenger server to earn the network fee themselves.
```

We should put an empty line between the title and the rest of the paragraph. Otherwise markdown renders them as a single paragraph.

**Original:**

![image](https://github.com/user-attachments/assets/0f2dfed2-2de9-47b3-8f2b-a8945749b1cb)

**With extra line:**

![image](https://github.com/user-attachments/assets/bd452c4c-87e0-4cbd-a138-5e298f4bbbba)

> Each message, that means each request and each response, requires the Network Fee. for example an OTA (distributor) sends a message request (0.01 CAM per message) to 10 Accommodation Suppliers. 7 send a respond message (each pay 0.01 CAM per message).

Reviewer: ```suggestion
Each message, that means each request and each response, requires the network fee. For example, an OTA (distributor) sends a message request to 10 Accommodation Suppliers, paying 0.10 CAM (0.01 CAM per message). If seven of them send a response back, each will pay 0.01 CAM per message.
```

The example was hard to understand. Is this better?

2. Camino Messenger Service Fee

>    The Service Fee is set by the supplier for each individual service. It is specified in CAM and can have any value including 0. It's purpose is an additional income stream, balancing out traffic/processing costs, replacement for booking fees or payment for providing information services. The Service Fee is split between the Supplier (70%) and Chain4Travel (30% for further development).

Reviewer: ```suggestion
2. Camino Messenger Service Fee

   The Service Fee is set by the supplier for each individual service. It is specified in CAM and can have any value including 0. It's purpose is an additional income stream, balancing out traffic/processing costs, replacement for booking fees or payment for providing information services. The Service Fee is split between the Supplier (70%) and Chain4Travel (30% for further development).
```

Same here, one extra empty line.

The Partner and Network messages will be made available to enable automatic detection of changed partners, their settings and fees.

> Figure 2: Camino Messenger Service Fee and Network Fee

Reviewer: This is confusing. I think it should say "Service Fee and Network Fee" only. 

If we put "Camino Messenger" before the Service Fee than it sounds like Network Fee is not related to Camino Messenger. Remember the last meetings we have, people already started to confuse "Network Fee" with the transaction fees you pay to the network.

I still think "Network Fee" is not a good term here. It should be "Messenger Fee" because it is paid to the messengers (Matrix Messenger operators).

Author: We can discuss to change the wording after TU2024 to avoid discrepancies between the different presentations and documentation (code)
